Install-Under-CentOS_LEMP.md

```diff
@@ -204,25 +204,45 @@ during the installation.
    provide recommendations based on the actual system which will be more
    tailored to your environment.
 
+   If using MariaDB less than version 11.1 or MySQL use the settings below.
+   Note that you should revise up or down your settings depending on
+   the Cacti recommendation that you see once you start the installer.
+
    ```shell
    [mysqld]
    character-set-server=utf8mb4
    collation-server=utf8mb4_unicode_ci
-   max_allowed_packet=18M
-   max_heap_table_size=98M
-   tmp_table_size=64M
-   join_buffer_size=64M
-   innodb_buffer_pool_size=488M
-   innodb_doublewrite=OFF
-   innodb_flush_log_at_timeout=3
-   innodb_read_io_threads=32
-   innodb_write_io_threads=16
+   innodb_file_format = Barracuda
+   max_allowed_packet = 16777777
+   join_buffer_size = 32M
+   innodb_file_per_table = ON
+   innodb_large_prefix = 1
+   innodb_buffer_pool_size = 250M
+   innodb_additional_mem_pool_size = 90M
+   innodb_flush_log_at_trx_commit = 2
    log-error                      = /var/log/mysql/mysql-error.log
    log-queries-not-using-indexes  = 1
    slow-query-log                 = 1
    slow-query-log-file            = /var/log/mysql/mysql-slow.log
```

Comment:
We should make this consistent, either with no spacing or make them aligned.

Reply:
done.